tictactoe.Tests/AiPlayerTest.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+using Xunit;
+
+namespace tictactoe.Tests
+{
+    public class AiPlayerTest
+    {
+        private Board board;
+        private MockConsoleInterface console;
+        private AiPlayer aiPlayer;
+
+        public AiPlayerTest()
+        {
+            console = new MockConsoleInterface();
+            board = new Board();
+            aiPlayer = new AiPlayer("O", console);
+        }
+
+        [Fact]
+        public void AiPlayerReturnsRandomNumberBetween1And9()
+        {
+            List<int> randomNumberList = new List<int>() { 1, 2, 3, 4, 5, 6, 7, 8, 9 };
+            int aiInput = aiPlayer.GetMove(board);
+            Assert.Contains(aiInput, randomNumberList);
```

Comment:
Something to think about: since `aiInput` can return a different value each time we run the test, it is hard to guarantee we will always have the same test result (ie, passing). Is there a way to ensure this test will pass all the time, and that we are delegating to a random function?

Reply:
An option would be to put a wrapper around Random and check if it was called